### DISCUSSION TITLE
part-ia/2012-8/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#427] [Question comments]
Part IA 2012 Differential Equations: Paper 2, Section II, A 

https://questions.tripos.org/part-ia/2012-8/

**Comment by hennalVEVO** [comment]
<details>
<summary>Solution outline</summary>

There is nothing clever about the method. Frobenius series about $x=0$.

You should get $\sigma = 1 \implies a_n = \frac{a_{n-2}}{n(n-1)}$ and $\sigma = 2 \implies a_n \frac{a_{n-2}}{n(n+1)}$. From here, once you work through the steps you should find that $y_1 = ax\operatorname{cosh}(x)$ and $y_2 = bx\operatorname{sinh}(x)$.

</details>
